app/src/main/res/layout/drawer_header.xml

```diff
@@ -28,41 +27,33 @@
         android:gravity="center"
         android:background="@color/primary_dark"/>
 
-
     <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:paddingTop="@dimen/toolbar_top_padding"
         android:layout_margin="16dp"
-        android:paddingBottom="8dp"
-        android:orientation="vertical"
-        android:layout_gravity="bottom"
```

Comment:
After remove this line, Maybe we have a chance to reduce the layout hierarchy. Replace the root viewgroup from FrameLayout to LinearLayout, and use padding to control children view position.